packages/@aws-cdk/aws-appmesh/lib/virtual-node.ts

```diff
@@ -216,7 +230,12 @@ export class VirtualNode extends VirtualNodeBase {
       resourceName: this.physicalName,
     });
   }
-
+  /**
+   * Adds Default Backend Configuration for virtual node to communicate with Virtual Services.
+   */
+  private setBackendDefaults(backendDefaults: ClientPolicy) {
```

Comment:
Same comment here as above:

1. Get rid of the `backendDefaults` field and the `setBackendDefaults` method, just make it a local variable of the constructor.
2. Why does `backendDefaults` accept a single `ClientPolicy`, while the L1 class accepts an array?

Reply:
I have created `backendDefaults` as a local variable of the constructor. The backendDefaults accepts an a single `ClientPolicy` instead of an array. ([Reference](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appmesh-virtualnode-backenddefaults.html))